ambry-api/src/main/java/com.github.ambry/router/GetBlobOptions.java

```diff
@@ -103,12 +103,21 @@ public int hashCode() {
     /**
      * Return blob info and blob data in the response.
      */
-    All, /**
+    All,
+
+    /**
      * Return just the blob data in the response.
      */
-    Data, /**
+    Data,
+
+    /**
      * Return just the blob info in the response.
      */
-    BlobInfo;
+    BlobInfo,
+
+    /**
+     * Return BlobInfo, but use information in the blob data to ensure that the fields are all populated correctly.
+     */
+    BlobInfoAll;
```

Comment:
do we really need this? It seems to be like this should be transparent behind `BlobInfo` (why would `BlobInfo` be used if the data could be incorrect?)

Reply:
Folded everything into `BlobInfo`